packages/components/src/utils/theme-variables.scss

```diff
@@ -6,3 +6,20 @@
 $components-color-accent: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
 $components-color-accent-darker-10: var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #006ba1));
 $components-color-accent-darker-20: var(--wp-components-color-accent-darker-20, var(--wp-admin-theme-color-darker-20, #005a87));
+
+// Used when placing text on the accent color.
+$components-color-accent-inverted: var(--wp-components-color-accent-inverted, $white);
+
+$components-color-background: var(--wp-components-color-background, $white);
+$components-color-foreground: var(--wp-components-color-foreground, $gray-900);
+
+// Used when placing text on the foreground color.
+$components-color-foreground-inverted: var(--wp-components-color-foreground-inverted, $white);
+
+$components-color-gray-100: var(--wp-components-color-gray-100, $gray-100);
```

Comment:
perhaps `scale` or `colorscale`?

Reply:
`background` (e.g. `background-100`) is probably the most correct and logical. Though I'm still leaning toward `gray` because the vast majority of consumers will be specifying a background color that is somewhere on the gray scale. Hard to decide!

Reply:
As far as we make sure we can tweak and change those variables names in the future (i.e. we take all precautions to avoid / limit their usage outside of the components package), when we can go ahead with `gray` for now and re-assess at a later point before marking he Theming APIs as "ready"